blueprints/addon/files/.travis.yml

```diff
@@ -13,8 +13,8 @@ sudo: false
 <% } %>
 env:
   # we recommend testing LTS's and latest stable release (bonus points to beta/canary)
-  - EMBER_TRY_SCENARIO=ember-lts-2.4
```

Comment:
do we want to remove 2.4 testing? that doesn't fit to comment above where I understand it as "testing all LTS's"